hijack/contrib/admin/apps.py

```diff
@@ -12,6 +12,11 @@ def ready(self):
         from . import HijackUserAdminMixin
 
         UserModel = get_user_model()
+
+        # User model may be a proxy and unregistered. See #328 for more details.
```

Comment:
Maybe add the full link to the issue, makes it easier to open the issue form code.